ui/src/routes/info/_layout.svelte

```diff
@@ -0,0 +1,171 @@
+<script>
+	import * as _ from 'lamb';
+	import {_screen}
+		from '@svizzle/ui/src/gauges/screen/ScreenGauge.svelte';
+
+	import ChevronLeft from '@svizzle/ui/src/icons/feather/ChevronLeft.svelte';
+	import ChevronRight from '@svizzle/ui/src/icons/feather/ChevronRight.svelte';
+	import Icon from '@svizzle/ui/src/icons/Icon.svelte';
+	import Link from '@svizzle/ui/src/Link.svelte';
+
+	import {isNotNil} from '@svizzle/utils';
+	import {toolName} from 'app/config';
+	import theme from 'app/theme';
+
+	const links = {
+		beta: 'Beta release',
+		privacy: 'Privacy',
+		disclaimer: 'Disclaimer',
+		feedback: 'Feedback',
+	}
+	const segments = _.keys(links);
+
+	export let segment;
+
+	$: currentValueIndex = _.findIndex(
+		segments,
+		_.is(segment)
+	);
+	$: prevSegment = segments[currentValueIndex - 1];
+	$: nextSegment = segments[currentValueIndex + 1];
+	$: hasPrevSegment = isNotNil(prevSegment);
+	$: hasNextSegment = isNotNil(nextSegment);
+</script>
+
+<svelte:head>
+	<title>EURITO - Info</title>
+	<meta
+		name='description'
+		content='All about accessibility in {toolName}, including a guide on how to enable the accessibility dialog, accessibility audit and other quality audits, plus some pointers to setup various accessibility tools on your system'
+	>
+</svelte:head>
+
+<main class={$_screen?.classes}>
+	<section>
+		<h2>Info</h2>
```

Comment:
This should be an `h1` like in the other text pages; text could be `App Information`